deploy/helm_charts/README.md

```diff
@@ -36,3 +36,29 @@ helm upgrade --install mixer-dev deploy/helm_charts/mixer \
     --set-file kgStoreConfig.bigqueryVersion=deploy/storage/bigquery.version \
     --set-file kgStoreConfig.bigtableImportGroupsVersion=deploy/storage/bigtable_import_groups.version
 ```
+
+## Releasing a helm chart
+
+Releasing a helm chart packages and uploads a helm chart into [AR(Artifact Registry)](https://cloud.google.com/artifact-registry/docs/overview) as a versioned artifact. Once released, the chart will be accessible by the public, given that the access to AR repository is configured.
+
+1.  Skip this step for upgrades. Make sure that an AR docker repository exists. If not, please follow the instructions [here](https://cloud.google.com/artifact-registry/docs/helm) to create one.
```

Comment:
Maybe add a little background for each bullet here. For example, 

1) I am not sure what "upgrades" here means.
3) should "mixer" be the only choice here?